app/common/models/game-session.coffee

```diff
@@ -3,6 +3,7 @@ mongoose = require 'mongoose'
 Schema = mongoose.Schema
 
 GameSessionSchema = Schema
+	game_master_id: Schema.Types.ObjectId
```

Comment:
One game master to rule 'em all?
I don't know, let's keep it like that for now?